README.md

```diff
@@ -57,16 +57,19 @@ celestia-appd --help
 ### Create your own single node devnet
 
 ```sh
-# WARNING: this deletes config, data, and keyrings from previous local devnets
-rm -r ~/.celestia-app
 
-# Start a single node devnet
+# Start a single node devnet using the pre-installed celestia app
 ./scripts/single-node.sh
 
+# Build and start a single node devnet
+./scripts/build-run-single-node.sh
```

Comment:
I'd be fine with only keeping the build and run, since I doubt anyone would need to rerun the script without building